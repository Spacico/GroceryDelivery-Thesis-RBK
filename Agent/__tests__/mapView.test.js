import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';
import MapView from 'react-native-maps';
import Mapo from '../Components/mapView';

describe('mapView Testing', () => {
    it('renders with out crashing', () => {
        const rendered = renderer.create(<Mapo />).toJSON();
        expect(rendered).toBeTruthy();
    });
})
//     it('contains TextInput', () => {
//         const TextInput = require('View');
//         const Tree = renderer.create(<View />).toJSON();
//         expect(Tree).toMatchSnapshot();
//     });
//     it('contains TouchableOpacity', () => {
//         const TouchableOpacity = require('MapView');
//         const Tree = renderer.create(<MapView />).toJSON();
//         expect(Tree).toMatchSnapshot();
//     });
//     it('contains View', () => {
//         const View = require('MapView.Marker');
//         const Tree = renderer.create(<MapView.Marker />).toJSON();
//         expect(Tree).toMatchSnapshot();
//     });
    
// })