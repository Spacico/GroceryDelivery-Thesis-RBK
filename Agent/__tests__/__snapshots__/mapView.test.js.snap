// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mapView Testing contains View 1`] = `
<AIRMapMarker
  image={undefined}
  onPress={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "position": "absolute",
      },
      undefined,
    ]
  }
/>
`;
